src/cr/cube/slices.py

```diff
@@ -266,7 +262,7 @@ def _slice(self):
 
     @lazyproperty
     def _transforms(self):
-        return Transforms(self._ordering, self._pruning, self._insertions)
+        return Transforms(self._slice, self._dimensions, self._pruning)
```

Comment:
I'd say let the Transforms object work out the value of `._pruning` for itself, now that it has the `._dimensions` (which is the authority). So that would mean delete `._pruning` property here and remove pruning param from Transforms constructor. Then think of who accesses pruning on a Transforms object and see if they can't get it for themselves as they likely have the dimension there, especially when pruning is dimension-by-dimension rather than all-or-nothing.

Also, shouldn't `._ordering` property come out too since this was the only caller?